apps/www/_blog/2024-12-06-restore-to-a-new-project.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Restore to a New Project'
+description: Effortlessly Clone Your Supabase Project
```

Comment:
```suggestion
description: Effortlessly Clone Data into Another Supabase Project
```

Reply:
Adjusted to `Effortlessly Clone Data into a New Supabase Project` as I don't want to indicate this can be any project, but a _new_ project.